### PR TITLE
[POC] Experimental Postgres support

### PR DESCRIPTION
**Pull request by eddyxu**
A proof of concept work to run Rikai in PostgreSQL.

### CODE REVIEW

File: experimental/pg/rikai/experimental/pg/model.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    # TODO: move load model into rikai core.
+    conf = {
+        "version": "1.0",
+        "name": f"load_{model_type}",
+        "flavor": flavor,
+        "modelType": model_type,
+        "uri": uri,
+    }
+    if uri:
+        spec = FileModelSpec(conf)
```
Comment (changhiskhan) [inline experimental/pg/rikai/experimental/pg/model.py:67]: should models just be stored in some pg metadata tables?

File: experimental/pg/sql/rikai--0.1.sql [hunk 2]
```diff
@@ -0,0 +1,61 @@
+-- Create Schema to contain all Rikai functionality
+CREATE SCHEMA IF NOT EXISTS ml;
+
+-- Semantic Types
+CREATE TYPE image AS (uri TEXT);
+
+CREATE TYPE detection AS (label TEXT, label_id int, box box, score real);
```
Comment (changhiskhan) [inline experimental/pg/sql/rikai--0.1.sql:7]: where's the box type?

File: experimental/pg/rikai/experimental/pg/model.py (hunk 1 above)
Comment (changhiskhan) [reply experimental/pg/rikai/experimental/pg/model.py:67]: oh hmmm actually if the model inf/train needs to be externalized then maybe not

File: experimental/pg/sql/rikai--0.1.sql (hunk 2 above)
Comment (eddyxu) [reply experimental/pg/sql/rikai--0.1.sql:7]: Box and Polygon `Mask in rikai` are already supported. https://www.postgresql.org/docs/current/datatype-geometric.html

File: experimental/pg/rikai/experimental/pg/model.py (hunk 1 above)
Comment (eddyxu) [reply experimental/pg/rikai/experimental/pg/model.py:67]: model specs are stored in `ml.models` table. It acts the same as in Spark's `CREATE MODEL /  SHOW MODELS`. 
So in this implementation, it treats `ml` schema / namespace as the ML catalog in Spark

**Comment by eddyxu** [comment]
Move the code to a separate repo https://github.com/eto-ai/rikai-pg,  giving its early maturity.

**Pull request closed without merging**